vm/systemSmartContracts/delegation.go

```diff
@@ -797,6 +821,13 @@ func (d *delegation) setAutomaticActivation(args *vmcommon.ContractCallInput) vm
 		return vmcommon.UserError
 	}
 
+	entry := &vmcommon.LogEntry{
```

Comment:
make a new function which creates this entry and adds it. Thus createAndAdd - you may get the args as vmcommon.ContractCallInput as input, plus [][]byte as topics

Reply:
and if input args is nil, maybe use args.Arguments.

Reply:
done.